components/SwipeListView.js

```diff
@@ -26,6 +26,11 @@ class SwipeListView extends Component {
             };
         }
     }
+    
+    componentWillReceiveProps(nextProps) {
```

Comment:
`componentWillReceiveProps` is deprecated. Could you use `componentDidUpdate` instead?

Reply:
Sure